docs/ibc/light-clients/proofs.md

```diff
@@ -1,3 +1,24 @@
 <!--
 order: 7
--->
\ No newline at end of file
+-->
+
+# Existence and Non-Existence Proofs 
+
+`ibc-go` packets are accompanied by proofs as defined in the IBC [ICS23](https://github.com/cosmos/ibc/tree/main/spec/core/ics-023-vector-commitments) specification, which is a general approach for verifying merkle trees.
+
+Currently, all Cosmos SDK modules (including `ibc-go`) contain their own stores, which maintain the state of the application module in an IAVL (immutable AVL) binary merkle tree format. The Cosmos SDK multistore therefore creates a simple merkle tree of all of these IAVL trees, and from each of these individual IAVL tree root hashes derives a root hash for the application state tree as a whole (the apphash).
+
+For the purposes of `ibc-go`, there are two types of proofs which are important: existence and non-existence proofs. Existence proofs are used for transactions which will result in the writing of a packet receipt into the IBC store on the receiving end of the transaction (ie: token transfers, connection handshakes), whereas non-existence proofs are used to timeout IBC packets.
```

Comment:
ditto with this line about packet specific wording. 

Should we also say somewhere that existence and non-existence terms are used interchangeably with membership and non-membership?